libs/ui/src/admin/ArtApprovalModel/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,243 @@
+                      <Center as={SplideSlide} key={image.id}>
+                        <ArtImage image={image} alt={art.title} />
+                      </Center>
+                    ))}
+                  </Splide>
+                ) : (
+                  <ArtImage image={art.images?.[0]} alt={art.title} />
+                )}
+              </Stack>
+              <VStack spacing={4} align={'start'} justify="space-between">
+                <VStack align={'start'} justify="space-between">
+                  <Flex align="start" justify="start" w="full" h={8}>
+                    <Text color={'blue.400'} fontWeight={'bold'}>
+                      {art?.title}
+                    </Text>
+                  </Flex>
+                  <HStack spacing={3} w={'full'}>
+                    {/* TODO art owner avatar should be here*/}
+                    <Avatar
+                      w={8}
+                      h={8}
+                      src="https://bit.ly/dan-abramov"
+                    ></Avatar>
+                    <Text fontWeight={'bold'}>{art?.artist?.name}</Text>
+                  </HStack>
+                  <Flex align="start" justify={'start'} w="full" h={'72px'}>
+                    {/* TODO scroll should be here when art has long text*/}
+                    <Text>{art?.content}</Text>
+                  </Flex>
+                </VStack>
+
+                <VStack align={'start'} w={'full'}>
+                  <Flex align="start" justify={'start'} w="full">
+                    <Text color={'black'} fontWeight={'bold'}>
+                      Give Feedback
+                    </Text>
+                  </Flex>
+
+                  <HStack w="full">
+                    {/* avatar*/}
+                    <Stack
+                      align={'start'}
+                      h={'136px'}
+                      flex={1}
+                      textAlign="center"
+                    >
+                      <Avatar
+                        w={8}
+                        h={8}
+                        src="https://bit.ly/dan-abramov"
+                        // src={user?.avatar}
+                      ></Avatar>
+                    </Stack>
+                    <VStack w="full">
+                      {/* text area, button group*/}
+                      <Stack w={'full'}>
+                        {/* text area*/}
+                        <Textarea
+                          size={['sm', 'md', 'lg', 'xl']}
+                          onChange={handleInputChange}
+                          placeholder={'type your comment here'}
+                        ></Textarea>
+                      </Stack>
+                      <Stack w={'full'}>
+                        {/*button group*/}
+                        <ButtonGroup
+                          alignItems="space-between"
+                          w={'full'}
+                          justifyItems="flex-end"
+                          spacing={[2, 4, 6, 8]}

```

Comment:
4 is enough. 6 and 8 seems a bit large to me.

```suggestion
                          spacing={4}

```